### PR TITLE
JIRA: ABC-105 Exported normals can't be read outside of Unity

### PR DESCRIPTION
**Pull request by clusty**
The bug was caused by the fact that the normals did not have the correct scope. All vertex attributes suffer from this issue:
UV1, UV2, Colours, Normals.

If number of Indices is the same as the number of faces * 3 (every point is used in only 1 face ), attributes are FaceVarying, else they are Vertex scope

### CODE REVIEW

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ## [2.2.0-exp.3] - 2021-01-22
 ### Changes
 - Minimum Unity version is 2019.4
+- Fixed a bug that caused the exported mesh normals to be unreadable in DCCs
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/CHANGELOG.md:5]: Just move to "Fixed" section + add a period.

**Review by sebastienduverne — COMMENTED** [review]
Minor changelog structure issue.

**Comment by clusty** [comment]
@cguer Please test that Maya is not complaining about exported mesh attributes: bonus points if whacky unity normals are preserved (could manufacture some normals by code if need be...).

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2020.2.3f1 (Windows only)

Validated:
- No attribute warning when importing alembic file into Maya
- "Messed up" normals in Unity are properly exported
- Works for tri and quad meshes, as well as sub-meshes with no connected vertices

(unrelated to this PR but also confirmed that quad polygons are preserved on export)

**Review by judubu — APPROVED** [review]
👍

**Pull request merged**